resources/assets/lib/chat/conversation-list.tsx

```diff
@@ -15,6 +15,7 @@ export default class ConversationList extends React.Component {
       <div className='chat-conversation-list'>
         {supportedChannelTypes.map((type) => (
           <React.Fragment key={type}>
+            {this.renderHeader(type)}
             {this.renderChannels(core.dataStore.channelStore.groupedChannels[type])}
             {this.renderSeparator()}
```

Comment:
looks like those can be combined into one function? And do the length check first thing and return null if empty (also removes need for most of separator hiding logic css except the last-child).